libcontainer/notify_linux_v2.go

```diff
@@ -0,0 +1,127 @@
+// +build linux
+
+package libcontainer
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"unsafe"
+
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sys/unix"
+)
+
+func getOOMCount(path string) (int, error) {
+	if _, err := os.Lstat(path); err != nil {
+		return 0, err
+	}
+
+	content, err := ioutil.ReadFile(path)
+	if err != nil {
+		return 0, err
+	}
+
+	oomCount := 0
+	lines := strings.Split(string(content), "\n")
+	for _, line := range lines {
+		arr := strings.Fields(line)
```

Comment:
`strings.Split(line, " ")` is 2x faster, if your input is standard (no extra spaces etc.), and in this case the input is standard.